### PR TITLE
Added facebook integration

### PR DESCRIPTION
**Pull request by nileshgulia1**
Fixes issue #798 

Changes: Added facebook section in settings so that people can post about susi.ai

Demo Link: https://nileshgulia1.github.io/chat.susi.ai/  after page loads ,please select chat with susi to view working demo

Screenshots for the change: 

screenshot from

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -25,9 +27,21 @@
     <meta name="twitter:url" content="http://chat.susi.ai">
     <meta name="twitter:title" content="Web Chat with Susi.ai - Open Source Personal Assistant">
     <meta name="twitter:description" content="Chat with an Open Source Personal Assistant">
+
```
Comment (isuruAb) [inline public/index.html]: empty lines ?

**Comment by isuruAb** [comment]
please check the travis failure

**Comment by nileshgulia1** [comment]
@isuruAb  fine i will figure this out , please let me know about bugs it has

File: public/index.html (hunk 1 above)
Comment (nileshgulia1) [reply public/index.html]: fine, i"ll remove those

**Comment by nileshgulia1** [comment]
hey @isuruAb  ,I don't know why travis is failing.can u please point out the particular file that is causing error,I will get it fixed.

**Comment by isuruAb** [comment]
@nileshgulia1  please check the travis report
https://travis-ci.org/fossasia/chat.susi.ai/builds/289625358?utm_source=github_status&utm_medium=notification

Your PR has lot of Trailing spaces and extra blank lines. please resolve those errors.

**Comment by nileshgulia1** [comment]
I will close the PR for now  and reopen it after removing errors.

**Pull request closed without merging**